### PR TITLE
Update _common.yaml

### PR DESCRIPTION
**Pull request by Kalma-House**
Synonyms added to request temperature.

### CODE REVIEW

File: sentences/es/_common.yaml [hunk 1]
```diff
@@ -51,6 +51,7 @@ expansion_rules:
   area: "[en | en el | en la | la | de la | del | de | de el | el] {area}"
   brightness: "{brightness} [porciento]"
   enciende: "(enciende | activa)"
+  pon: "(pon | poner | sube | subir | baja | bajar | ajusta | ajustar | configura | configurar)"
```
Comment (cibernox) [inline sentences/es/_common.yaml:54]: Does it make sense to include the infinitives here? Personally I don't think anyone would talk like that.

Comment (Kalma-House) [reply sentences/es/_common.yaml:54]: I would not talk like that, but i know some people who do. Each person has a digfferent style of talking to computers.

Comment (mardito) [reply sentences/es/_common.yaml:54]: Yes, for example my wife say "subir persianas de comedor".

**Review by mardito — APPROVED** [review]
LGTM

**Review by cibernox — APPROVED** [review]

File: sentences/es/_common.yaml (hunk 1 above)
Comment (cibernox) [reply sentences/es/_common.yaml:54]: I stand corrected then 🤷‍♂️

**Comment by davefx** [comment]
If I'm not wrong, these changes have already been added

**Comment by Kalma-House** [comment]
Yae, someone did. Thus i am closing it.

**Pull request closed without merging**